analysis/preprocess/preprocess_data.R

```diff
@@ -67,7 +67,8 @@ message ("Cohort ",cohort_name, " description written successfully!")
 
 df$cov_bin_obesity <- ifelse(df$cov_bin_obesity == TRUE | 
                                df$cov_cat_bmi_groups=="Obese",TRUE,FALSE)
-df[,c("cov_num_bmi")] <- NULL
+# WE WANT TO KEEP THIS FOR TABLE 1
+# df[,c("cov_num_bmi")] <- NULL
```

Comment:
By removing this variable, stage2_missing table shows an Error:
```
Error in `dplyr::select()`:
! Can't subset columns that don't exist.
✖ Column `cov_num_bmi` doesn't exist.
```

Reply:
That's why it's been commented out because we don't want to remove it. Have you deleted the '#' ?I've pushed the change. Please pull and rerun

Reply:
Please make sure you do a full re-run from study def's too so that you have the most up to date files from the repo